src/current/v24.2/changefeed-sinks.md

```diff
@@ -212,6 +212,54 @@ See the [Changefeed Examples]({% link {{ page.version.version }}/changefeed-exam
 
 {% include {{ page.version.version }}/cdc/note-changefeed-message-page.md %}
 
+## Amazon MSK
+
+{{site.data.alerts.callout_info}}
+On CockroachDB {{ site.data.products.core }} clusters, you must create instances in the same VPC as the MSK or MSK Serverless cluster in order for the changefeed to authenticate successfully.
+
+If you would like to connect a CockroachDB {{ site.data.products.dedicated }} cluster to an Amazon MSK cluster, contact your Cockroach Labs account team.
+{{site.data.alerts.end}}
+
+Changefeeds can deliver messages to Amazon MSK clusters ([Amazon Managed Streaming for Apache Kafka](https://docs.aws.amazon.com/msk/latest/developerguide/what-is-msk.html)). Amazon MSK cluster types include: [MSK](https://docs.aws.amazon.com/msk/latest/developerguide/create-cluster.html) and [MSK Serverless](https://docs.aws.amazon.com/msk/latest/developerguide/serverless.html). Changefeeds support the following authentication methods for these MSK cluster types:
+
+- MSK: `SCRAM` or `IAM`
+- MSK Serverless: `IAM`
+
+{% include_cached new-in.html version="v24.2" %} Changefeeds can deliver messages to MSK and MSK Serverless clusters using AWS IAM roles.
+
+{% comment %}will change out the links here to CRDB tutorials in a follow-up PR.{% endcomment %}
+
+For initial setup guides, refer to the AWS documentation:
+
+- [Getting started using Amazon MSK](https://docs.aws.amazon.com/msk/latest/developerguide/getting-started.html)
+- [Getting started using MSK Serverless clusters](https://docs.aws.amazon.com/msk/latest/developerguide/serverless-getting-started.html)
+
+To connect to an MSK cluster using `SCRAM` authentication, you must include the following parameters in the URI:
+
+~~~
+kafka://{cluster_endpoint}/?tls_enabled=true&sasl_enabled=true&sasl_mechanism=SCRAM-SHA-512&sasl_user={user}&sasl_password={password}
+~~~
+
+To connect to an MSK or MSK Serverless cluster using AWS IAM roles, you must include the following parameters in the URI:
+
+~~~
+kafka://{cluster_endpoint}/?tls_enabled=true&sasl_enabled=true&sasl_mechanism=AWS_MSK_IAM&sasl_aws_region=us-east-2&sasl_aws_iam_role_arn={arn}&sasl_aws_iam_session_name={your_session_name}
+~~~
+
+Changefeeds connecting to Amazon MSK clusters use the `kafka://` scheme.
+
+URI Parameter  | Description
+---------------+------------------------------------------------------------------
+`cluster_endpoint` | The endpoint listed for your Amazon MSK cluster in the AWS Console. For example, `boot-a1test.c3.kafka-serverless.us-east-2.amazonaws.com:9098`.
+`sasl_aws_iam_role_arn` | The ARN for the IAM role that has the permissions to create a topic and send data to the topic.
+`sasl_aws_iam_session_name` | The user-specified string that identifies the session in AWS.
+`sasl_aws_region` | The region of the Amaozn MSK cluster.
+`sasl_enabled` | Set to `true`.
```

Comment:
This is unclear to me; does it mean that it's `true` by default or that it must be set to `true`?

Should/can it be written in a way that's consistent with the `tls_enabled` description?

```suggestion
`sasl_enabled` | Set to `true`.
```